uk_bin_collection/uk_bin_collection/councils/CastlepointDistrictCouncil.py

```
@@ -53,68 +55,57 @@ def parse_data(self, page: str, **kwargs) -> dict:
-            )
-            month = datetime.strptime(month_txt, "[%b]").strftime("%m")
-
-            # e.g. "About my Street - August 2025"
-            year_txt = soup_one_month.find("h1").get_text(strip=True)
-            year = datetime.strptime(year_txt, "About my Street - %B %Y").strftime("%Y")
+        for container in calendarContainer2:
+            table = container.find("table", class_="calendar")
+            if not table:
+                continue
+            month_txt = container.find("tr", class_="calendar").get_text(strip=True)
+            month = datetime.strptime(month_txt, "%B").strftime("%m")
```

Ruff (0.14.10)

`datetime` may be undefined, or defined from star imports

(F405)